mappings/net/minecraft/network/packet/s2c/play/WorldBorderUpdateS2CPacket.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_5889 net/minecraft/network/packet/s2c/play/WorldBorderUpdateS2CPacket
```

Comment:
This is the world border *initialize* packet

Reply:
k